app/controllers/api/account.php

```diff
@@ -1677,6 +1677,96 @@
         ])]), Response::MODEL_JWT);
     });
 
+App::post('/v1/account/targets')
```

Comment:
Lets make these routes only for push targets, like `/v1/account/targets/push`, these will be the only type needed to be create/updated by clients